synapse/models/orgs.py

```diff
@@ -484,8 +484,11 @@ def getModelDefs(self):
                         'doc': 'The org which issues id numbers of this type.',
                     }),
                     ('name', ('str', {}), {
+                        'alts': ('names',),
```

Comment:
We recently settled on the idea that "id" in docs should be capitalized. Would you mind mopping up this doc for consistency with the new one?